test/e2e/e2e_test.go

```diff
@@ -30,6 +34,10 @@ const (
 	ciNetworkName        = "kind"
 )
 
+var (
+	agnhostImage = imageutils.GetE2EImage(imageutils.Agnhost)
```

Comment:
I start to think that we should hardcode and not depend on the framework, what do you think?
```
k8s.gcr.io/e2e-test-images/agnhost:2.26
```

you already felt the pain of dealing with these images upstream